people/will/homework/hw_1.html

```diff
@@ -7,30 +7,32 @@
 
 <!-- Testing page to try out if and else statements-->
 
+
+<h1 style="text-align: center"> The Ultimate Source for Finding <span style = "color: #f00"> Power Level</span></h1>
+
+
+<form style = "text-align: center; margin-top: 50px;" name = "power1" action = "#">What is your power level? 
+	<input type="text" id = "powerInput" name="text1"><input type="button" name="button1" value="Check power" onclick="powerCheck()">
+</form>
+
+<p style = "margin-top: 200px; text-align: center;">This took <em>far</em> longer than I wish it had.</p>
+
 <script type="text/javascript">
 	
+	var powerLevel = document.getElementById('powerInput');
```

Comment:
Thanks @garvank . The updates should be reflected now, and it makes sense as to why. If I were to want to use that variable in other functions, I would want to keep it outside.